.github/workflows/nomad-integration-test.yaml

```diff
@@ -5,6 +5,7 @@ jobs:
     env:
       TEST_PLATFORM: nomad
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
Setting job timeout since `nomad run` doesn't have a timeout to avoid wasting free minutes.